### PR TITLE
Update documentation for ISS

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
## Proposed change

Part of the fix [for](https://github.com/home-assistant/core/issues/76615). See [this](https://github.com/home-assistant/core/pull/77218) issue for information.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/77218
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Please rebase your branch on latest `next` branch and remove the unrelated changes.

File: source/_integrations/iss.markdown [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: International Space Station (ISS)
-description: Know if or when ISS will be above your home location
+description: Will show you the ISS location and how many people are in space
```
Comment (MartinHjelmare) [inline source/_integrations/iss.markdown]:
```suggestion
description: Show the ISS location and how many people are in space
```

**Comment by DurgNomis-drol** [comment]
Should be ready for review

File: source/_integrations/iss.markdown [hunk 2]
```diff
@@ -16,11 +16,9 @@ ha_integration_type: integration
 
 The `iss` platform uses the
```
Comment (MartinHjelmare) [inline source/_integrations/iss.markdown]:
```suggestion
The International Space Station (ISS) integration uses the
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**